dashboard/src/pages/packages/index.vue

```diff
@@ -103,19 +108,60 @@ const toggleLegend = () => {
         </tbody>
       </table>
     </div>
-    <div>
-      <nav aria-label="Package list pages">
+    <div v-if="packageStore.pager.total > 1">
+      <nav role="navigation" aria-label="Pagination navigation">
         <ul class="pagination justify-content-center">
+          <li v-if="packageStore.pager.total > packageStore.pager.maxPages">
+            <a
+              href="#"
+              :class="{
+                'page-link': true,
+                disabled: packageStore.pager.current == 1,
+              }"
+              aria-label="Go to first page"
+              @click.prevent="packageStore.fetchPackages(1)"
+            >
+              <IconSkipStartFill /> First
+            </a>
+          </li>
           <li class="page-item">
             <a
               href="#"
               :class="{
                 'page-link': true,
                 disabled: !packageStore.hasPrevPage,
               }"
+              aria-label="Go to previous page"
               @click.prevent="packageStore.prevPage"
-              >Previous</a
             >
+              <IconCaretLeftFill /> Previous
```

Comment:
Similar to the previous and next links when you have those page links beside the current one.

Reply:
I don't understand what you mean here.

Reply:
I meant that, if you are on page 2, you always have the page 1 and page 3 links, which make the next/previous links kind of unnecessary. I think it looks worst without them, so not a big deal.